src/all.js

```diff
@@ -17,8 +17,11 @@ function initAll (options) {
   // Defaults to the entire document if nothing is set.
   var scope = typeof options.scope !== 'undefined' ? options.scope : document
 
-  // Find all buttons with [role=button] on the scope to enhance.
-  new Button(scope).init()
+  // Find all links with role="button" and all buttons to enhance.
+  var $buttons = scope.querySelectorAll('a[role="button"], button')
```

Comment:
I think you could remove the `a` since `[role="button"]` on it's own would be fine.

Would this need to include `input` elements that are buttons?

When I initially did this work I was worried that we'd end up initialising this JavaScript on a massive amount of buttons if it were on a case working system screen for example.

Should we be thinking about the performance impact of this?

Reply:
Right, input is a valid case too, I've updated to contain it.

I share your concerns, but the current solution (which I consider smart) doesn't seem to work in all cases. And thanks for sharing the initial work, seems to be very considerate.

Would you mind dismissing the initial review, since the following updates are slightly expanding outside the initial scope?

Reply:
We could also consider removing the feature, not likely we'd want to do this but it's an option.